# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../evas/PATH_MAX.patch
# Copyright (C) 2018 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- evas-1.7.10/src/bin/evas_cserve2_shm.c.vanilla	2018-11-17 21:52:44.079999931 +0000
+++ evas-1.7.10/src/bin/evas_cserve2_shm.c	2018-11-17 21:53:29.779999930 +0000
@@ -8,6 +8,7 @@
 #include <sys/mman.h>
 #include <sys/stat.h>
 #include <fcntl.h>
+#include <sys/param.h>
 #include <unistd.h>
 
 struct _Shm_Mapping
